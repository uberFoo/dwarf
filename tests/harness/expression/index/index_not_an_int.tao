fn main() {
    let a = "Hello, world!";
    let b = a["a"];
}
