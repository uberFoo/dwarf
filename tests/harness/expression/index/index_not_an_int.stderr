[31mError:[0m Type mismatch: expected `int`, found `string`.
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0mindex_not_an_int:3:15[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m3 â”‚[0m [38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249ml[0m[38;5;249me[0m[38;5;249mt[0m[38;5;249m [0m[38;5;249mb[0m[38;5;249m [0m[38;5;249m=[0m[38;5;249m [0m[38;5;249ma[0m[38;5;249m[[0m[33m"[0m[33ma[0m[33m"[0m[38;5;249m][0m[38;5;249m;[0m
 [38;5;240m  â”‚[0m               [33mâ”€[0m[33mâ”¬[0m[33mâ”€[0m  
 [38;5;240m  â”‚[0m                [33mâ•°[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m expected [36mint[0m
 [38;5;240m  â”‚[0m                [31mâ”‚[0m   
 [38;5;240m  â”‚[0m                [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m found [36mstring[0m
[38;5;246mâ”€â”€â”€â•¯[0m

